packages/core/contracts/financial-templates/common/FeePayer.sol

```diff
@@ -11,14 +11,15 @@ import "../../common/implementation/Testable.sol";
 import "../../oracle/interfaces/StoreInterface.sol";
 import "../../oracle/interfaces/FinderInterface.sol";
 import "../../oracle/implementation/Constants.sol";
+import "./FinancialContractInterface.sol";
 
 /**
  * @title FeePayer contract.
  * @notice Provides fee payment functionality for the ExpiringMultiParty contract.
  * contract is abstract as each derived contract that inherits `FeePayer` must implement `pfc()`.
  */
 
-abstract contract FeePayer is Testable, Lockable {
+abstract contract FeePayer is FinancialContractInterface, Testable, Lockable {
```

Comment:
`FundingRateStore` needs access to `pfc()`, so I added an interface it can access instead of importing this entire `FeePayer` contract